authors/apps/comments/views.py

```diff
@@ -95,32 +101,38 @@ def patch(self, request, pk):
         self.get_author_profile(request)
         if not self.is_author:
             return Response({
-                    "response": "You can only edit a comment you authored",
-                            "status_message": "Failed: Access denied"},
-                            status=status.HTTP_403_FORBIDDEN)
+                "response": "You can only edit a comment you authored",
+                "status_message": "Failed: Access denied"},
+                status=status.HTTP_403_FORBIDDEN)
         serializer = self.serializer_class(
-            self.comment, self.data, partial=True)
-        serializer.is_valid(raise_exception=True)
+            self.comment,
+            self.data,
+            partial=True,
+            context={'article': self.article}
+        )
+        serializer.is_valid(raise_exception=True,
+                            message="Could not update comment")
         serializer.save()
         return Response({"comment": serializer.data,
-                    "message": "Successfully edited comment: {}".format(pk)},
+                         "message": "Successfully edited comment: {}".format(
```

Comment:
Same case here, the user doesn't want to know they edited comment 1

Reply:
Same as for this.